src/SerialPort/ConflictingSettingsDialog.tsx

```diff
@@ -150,30 +156,21 @@ const DisplayConflictingSettings = ({
         : [];
 
     return (
-        <div
-            style={{
-                width: '100%',
-                display: 'flex',
-                justifyContent: 'flex-start',
-                margin: '32px 0',
-            }}
-        >
-            <div style={{ marginRight: '128px' }}>
-                <b>Active settings</b>
+        <div className="tw-preflight tw-my-4 tw-flex tw-w-full tw-justify-start">
+            <div className="tw-pr-7">
                 {activeSettings ? (
-                    <ul style={listStyle}>
+                    <ul>
+                        <li className="tw-p-1 tw-font-bold">Active settings</li>
                         {allKeys.map(key => (
                             <li
                                 key={key}
-                                style={{
-                                    ...listItemStyle,
-                                    backgroundColor:
-                                        conflictingSettings.includes(key)
-                                            ? '#FFCDD2'
-                                            : undefined,
-                                }}
+                                className={`tw-p-1 ${classNames(
+                                    conflictingSettings.includes(key)
+                                        ? 'tw-bg-orange-100'
+                                        : 'tw-bg-green-100'
+                                )}`}
                             >
-                                {key}:
+                                <span className="tw-font-semibold">{`${key}: `}</span>
```

Comment:
```suggestion
                                <span className="tw-font-semibold">{key}:</span>
```
might require eslint check as well